.github/workflows/cml-with-dvc.yaml

```diff
@@ -19,25 +18,31 @@ jobs:
         run: |
           cml runner \
               --cloud=aws \
-              --labels=cml-runner \
               --cloud-region=eu-west \
               --cloud-type=t2.micro \
+              --labels=cml-runner \
               --single
   train-model:
     needs: deploy-runner
     runs-on: [self-hosted, cml-runner]
     timeout-minutes: 120 # 2h
-    container:
-      image: docker://iterativeai/cml:0-dvc2-base1
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - uses: iterative/setup-cml@v1
       - name: Train model
         env:
           REPO_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
-          GDRIVE_CREDENTIALS_DATA: ${{ secrets.GDRIVE_CREDENTIALS_DATA }}
         run: |
           cml ci
           pip install -r requirements.txt
+          # should this be `dvc exp run/repro`?
+          python get_data.py
           python train.py
```

Comment:
not sure about this

Reply:
I haven't defined a pipeline in DVC, so I don't think this would work out of the box. In this guide, I'm so far only using the data versioning capability of DVC. Might be worth expanding upon it, but that would also blow up the scope of the blogpost. I think I'll settle for a "next up" in the conclusions.